modules/olm-installing-opm.adoc

```diff
@@ -14,9 +14,10 @@ endif::[]
 
 You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
 
-.Prerequisites
+.Prerequisites for Linux
```

Comment:
```suggestion
.Prerequisites

* If you use a Linux as your workstation operating system, it must use the following packages:
** `podman` version 1.4.4+ (version 2.0+ recommended)
** `glibc` version 2.28+
```
This is a standard section title, so we shouldn't change it. I think we can move the information down, like this. This will be a better way to manage operating system-specific requirements they emerge for other platforms later.

Reply:
@kalexand-rh Thank you for your review and kind point. I've modified it as you mentioned.